tasks/update-code/git.yml

```diff
@@ -29,6 +29,10 @@
   register: ansistrano_git_result
   when: ansistrano_git_identity_key_path|trim != ""
 
+- name: ANSISTRANO | GIT | Shred GIT deployment key
+  command: "shred -f {{ ansistrano_deploy_to }}/git_identity_key"
```

Comment:
Sorry to be this picky, but just in case... shouldn´t it be?

```
command: shred -f "{{ ansistrano_deploy_to }}/git_identity_key"
```

I am sure your stuff will work but just for coherence with other commands we have